kalite/coachreports/features/environment.py

```diff
@@ -1,4 +1,4 @@
 """
 environment.py specific to the this app
 """
-from kalite.testing.base_environment import before_all, after_all, before_feature, after_feature
+from kalite.testing.base_environment import *
```

Comment:
@rtibbles `import *` is never a good idea when you are importing from a module that doesn't define `__all__`